lib/report_portal/cucumber/parallel_report.rb

```diff
@@ -86,14 +78,21 @@ def done(desired_time = ReportPortal.now)
 
       private
 
-      def file_with_launch_id
-        Pathname(Dir.tmpdir) + "parallel_launch_id_for_#{@pid_of_parallel_tests}.lck"
+      def lock_file(file_path = nil)
+        file_path ||= Dir.tmpdir + "parallel_launch_id_for_#{@pid_of_parallel_tests}.lock"
+        super(file_path)
       end
 
       def set_parallel_tests_vars
         pid = Process.pid
         loop do
+          #FIXME failing in jenkins
```

Comment:
What does this comment mean?

Reply:
Jenkins slave failing to access proc table with ruby 2.3, need to investigate why